app/javascript/src/Settings/authentication-widget.jsx

```diff
@@ -37,6 +39,18 @@ const setValue = (el, val) => {
 }
 const setInputValue = (val) => (isReadOnly) => isReadOnly ? setting => setValue(setting, val) : setting => setting
 
+const clearPublicURLs = () => {
+  document.querySelector('#service_proxy_attributes_sandbox_endpoint').value = ''
+  document.querySelector('#service_proxy_attributes_endpoint').value = ''
+}
+
+const confirmDeploymentOptionChange = (isHostedToSelfManaged) => {
+  const message = isHostedToSelfManaged
+    ? 'This action deletes the corresponding routes in OpenShift. Do you want to continue?'
```

Comment:
```suggestion
    ? 'This will delete the existing Public Base URLs upon form submission. Do you want to continue?'
```

I don't think we need technical terms like routes and OpenShift. This way it's also valid for SaaS.